### ISSUE TITLE
Publish to open-vsx.org?

### ISSUE THREAD

**Issue by hl2guide** [tierratelematics/code-beautifier#45]
Please consider publishing this extension to: https://open-vsx.org/

So that users of VSCodium can use it too.

**Comment by valajczech** [comment]
Yeah, please. Company I'm working in uses this as it's default SCSS formatter but I cannot use it since I'm using VSCodium.
